spec/BowlingSpec.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable prefer-arrow-callback */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-undef */
+// eslint-disable-next-line no-undef
+// eslint-disable-next-line prefer-arrow-callback
+describe('Bowling', function () {
+  let bowling;
+
+  beforeEach(function () {
+    bowling = new Bowling();
+  });
+
+      it('starts as an empty scorecard', function () {
+      expect(bowling._rolls).toEqual([]);
+      });
+
+      it('can score a gutter game', function () {
+        for (let i = 0; i < 20; i++) {
+        bowling.roll(0);
+        }
+        expect(bowling.score()).toEqual(0);
+      });
+
+      it('can score a whole game of ones', function () {
+        rollMany(1, 20);
+        expect(bowling.score()).toEqual(20);
+      });
+
+      // eslint-disable-next-line no-var
+      // eslint-disable-next-line vars-on-top
+      var rollMany = function (pinsDown, rolls) {
```

Comment:
I like the inline function but perhaps it could have been used in the previous test too.